client/src/ui/css/App.css

```diff
@@ -170,15 +164,10 @@
   background-color: #f4f9fe;
 }
 
-/* I believe this is only used in recentreview.jsx so will remove soon TODO */
-.container {
-  background-color: rgba(255, 255, 255, 0);
-}
-
 .sign-in-button {
   position: absolute;
-  right: 80px;
-  top: 50px;
+  right: 30px;
```

Comment:
Is there a reason for this sign in button change?

Reply:
On mobile, the sign-in button looked jank (like near the middle of the screen) rather than flushed to the side. 

I did this as a sort of temporary fix as I think it would look better if the sign in buttons were consistently flushed to the side.

I had the right for 30px as the margin-right of the search bar was also 30px (but doesn't exactly line up using 30px right). 

Changed it to like ~15px right. 

**Before:**
<img width="501" alt="image" src="https://github.com/cornell-dti/course-reviews-react-2.0/assets/46132945/fce95a2d-7b97-47fd-af04-df8eeda83c54">

**After:**
<img width="501" alt="image" src="https://github.com/cornell-dti/course-reviews-react-2.0/assets/46132945/313cf0b6-37d9-4bad-a675-e2af21dff7e7">

Reply:
Oh yes perfect! Thank you so much! This was one of the bugs I wanted to fix 👍💯